package.json

```diff
@@ -71,19 +72,16 @@
     "smooth-scroll": "^16.1.3",
     "store": "^2.0.12",
     "undertaker-forward-reference": "^2.0.0",
-    "webpack": "^5.74.0",
```

Comment:
Is this best to keep in? While not explicitly referenced, we depend on the webpack version for the config files (`webpack.config.js`)